src/web/templates/user_rotator_card.html

```diff
@@ -15,31 +15,37 @@
 
 {% block card_text %}
     <div>
-        Délai de rotation : {{ rotator.delay }} sec.
+        {{ _('Rotation delay: %(seconds)d sec.', seconds=rotator.delay) }}
     </div>
     <div class="mb-2">
         {% if not rotator.families and not rotator.screens %}
-            Aucun écran ni famille à afficher.
+            {{ _('No screen neither family to rotate.') }}
```

Comment:
```suggestion
            {{ _('No screen nor family to rotate.') }}
```
But we might need to correct a bunch of stuff to have less weird grammar